client2/.eslintrc.js

```diff
@@ -88,6 +88,13 @@ module.exports = {
             4,
         ],
         "react/forbid-prop-types": "off",
+        "import/extensions": 0,
```

Comment:
Most of these are temporary.
I did disable some linting rules because I'm being bombarded with errors.